src/Tizen.NUI/src/public/BaseComponents/ControlState.cs

```diff
@@ -138,7 +127,8 @@ public static ControlState Create(params ControlState[] states)
             if (states.Length == 1)
                 return states[0];
 
-            ControlState newState = new ControlState();
+            var newState = new ControlState(0UL);
```

Comment:
Unlike to NUI2, the `ControlState` is class type so it creates temporary `ControlState` object on heap when operating `+=`.
How about this?
```C#
public static ControlState Create(params ControlState[] states)
{
    ulong newbits = 0UL;

    for (int i = 0; i < states.Length; i++)
    {
        newbits |= states[i].bitFlags;
    }

    return new ControlState(newbits);
}
```